packages/vite/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@redwoodjs/vite",
+  "version": "3.2.0",
+  "description": "Vite configuration package for Redwood",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/vite"
+  },
+  "license": "MIT",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
+  "bin": {
+    "vite": "./bins/vite.mjs",
+    "vite__": "./bins/vite.mjs"
+  },
+  "scripts": {
+    "build": "yarn build:js",
+    "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\""
+  },
+  "dependencies": {
+    "@babel/runtime-corejs3": "7.20.6",
+    "@esbuild-plugins/node-globals-polyfill": "0.1.1",
+    "@originjs/vite-plugin-commonjs": "1.0.3",
+    "@redwoodjs/internal": "3.2.0",
+    "@vitejs/plugin-react": "2.2.0",
+    "core-js": "3.26.1",
+    "vite": "3.2.5",
+    "vite-plugin-html": "3.2.0"
+  },
```

Comment:
`esbuild` is missing given it's used in the `src/index.ts` file (line 8):

```ts
import { transform } from 'esbuild'
```

Reply:
Esbuild is a dependency of vite, should be ok no?

Reply:
It may be ok, but it may also be susceptible to hoisting since the internal package lists esbuild as a dependency. I'm just going off of yarn's rulebook which says "Packages should only ever require what they formally list in their dependencies" https://yarnpkg.com/advanced/rulebook#packages-should-only-ever-require-what-they-formally-list-in-their-dependencies